site/profile/manifests/core/ifdown.pp

```diff
@@ -0,0 +1,17 @@
+# @summary
+#   Special case for servers that do PXE from one interface and change
+#   to another one after the first puppet run  IT-2199
+
+class profile::core::ifdown (
+  $interface,
+)
+{
+  $command = "ifconfig ${interface} down"
```

Comment:
It would be safer to use the `ip` command as `ifconfig` isn't part of the el7 @core.  I just happen to have added it to the rke role.

Reply:
changed to ip instead of ifconfig